crates/voicevox_core/src/c_export.rs

```diff
@@ -254,13 +257,113 @@ pub extern "C" fn voicevox_load_openjtalk_dict(dict_path: *const c_char) -> Voic
     result_code
 }
 
+#[no_mangle]
+pub extern "C" fn voicevox_audio_query(
+    text: *const c_char,
+    speaker_id: i64,
+    output: *mut *mut c_char,
```

Comment:
```suggestion
    output_audio_query_json: *mut *mut c_char,
```

名前を書いてあげたほうが伝わると思います

Reply:
確かにそうですね